src/test/java/org/kainos/ea/service/JobRolesDataTest.java

```diff
@@ -30,7 +30,21 @@ public class JobRolesDataTest {
     void getJobRoles_shouldReturnJobRoles_whenJobsReturnsJobRoles () throws SQLException, DatabaseConnectionException{
         List<JobRolesResponse> expectedResult = new ArrayList<>();
 
-        JobRolesResponse j = new JobRolesResponse(1, "Software Engineer", "Engineering", "Trainee");
+        JobRolesResponse j = new JobRolesResponse(1, "Software Engineer", "Engineering", "Trainee", 1, 1);
+        expectedResult.add(j);
+
+        Mockito.when(databaseConnector.getConnection()).thenReturn(conn);
+        Mockito.when(jobRolesData.getJobRoles(conn)).thenReturn(expectedResult);
+        List<JobRolesResponse> result = jobsService.getJobRoles();
+
+        assertEquals(expectedResult, result);
+    }
+
+    @Test
+    void getJobRoles_shouldReturnBandAndCapability_whenJobsReturnsBandAndCapability () throws SQLException, DatabaseConnectionException{
```

Comment:
This is testing a different scenario, the one I talk about in your comment on JobRolesData.java, however like I mentioned if you only want me to only return capabilities and bands that have current job roles against them, and not just all capabilities and bands, then this can be done too